File: CSharp/HtmlTest/Base/CloneTestWalker.cs
using HtmlWalker;
using HtmlWalker.Platform;
using System;

namespace HtmlTest.Base
{
    public class CloneTestWalker : TestWalker
    {
        public CloneTestWalker(WalkerPlatform platform, WalkerPlatform.IOptions options = null)
            : base(platform, options) => Walkers.Add(new CloneWalker(Output));
    }
}

File: Java/htmlwalker/src/test/java/htmltest/base/CloneTestWalker.java
package htmltest.base;

import htmlwalker.CloneWalker;
import htmlwalker.platform.WalkerPlatform;

public class CloneTestWalker extends TestWalker
{
    public CloneTestWalker(WalkerPlatform platform, WalkerPlatform.IOptions options)
    {
        super(platform, options);
        walkers.add(new CloneWalker(output()));
    }

    public CloneTestWalker(WalkerPlatform platform)
    {
        this(platform, null);
    }
}
